components/slider/style/index.less

```diff
@@ -2,20 +2,11 @@
 @import "../../style/mixins/index";
 
 @slider-prefix-cls: ~"@{ant-prefix}-slider";
-// slider color
-@slider-disabled-color: @disabled-color;
-// tooltip
-@slider-tooltip-color: #fff;
```

Comment:
You shouldn't change the original var name which would cause a break change.

Reply:
Sorry, but I couldn't see that `@slider-tooltip-color`, `@slider-tooltip-bg`, `@slider-tooltip-arrow-width`, `@slider-tooltip-distance` and`@slider-tooltip-arrow-color` are being referenced anywhere so I thought it safe/good to remove them.

Reply:
You are right, though `@slider-margin` is used before.

Reply:
Ahh yes, added it back in https://github.com/ant-design/ant-design/pull/9065/commits/56c1ca9025ce02c390bfaa55e201dada5b9df5f1